src/components/Trade/TradeInput.tsx

```diff
@@ -33,16 +33,44 @@ import { firstNonZeroDecimal } from 'lib/math'
 
 const FiatInput = (props: InputProps) => (
   <Input
+    minHeight={55}
+    id='tradevalue'
     variant='unstyled'
     size='xl'
     textAlign='center'
-    fontSize='5xl'
+    fontSize={getFontSizeT()}
     mb={6}
     placeholder='$0.00'
     {...props}
   />
 )
 
+var valueTrade: number | undefined
+
+const getFontSizeT = () => {
```

Comment:
same comment as above. either way we should be using chakra-ui theme sizes instead of pixel sizes.

Reply:
Chakra theme size has a large difference margin, xl and 2xl (a font in 2xl before reaching xl would clip)